### ISSUE TITLE
uncalled list-ports - nothing listed

### ISSUE THREAD

**Issue by vincentmanz** [skovaka/UNCALLED#4]
Hi, 

I have installed UNCALLED and [Read Until Api](https://github.com/nanoporetech/read_until_api) 
following the recommendations. 
```
/opt/ont/minknow/ont-python/bin/python setup.py build_ext --library-dirs /usr/lib/x86_64-linux-gnu/hdf5/serial/lib/ --include-dirs /usr/lib/x86_64-linux-gnu/hdf5/serial/include
sudo /opt/ont/minknow/ont-python/bin/python setup.py install 

```
However, when I am trying to look for the port nothing is happening. 

`uncalled list-ports --log-dir /var/log/minknow/`

Do you have any idea how to get it to work?

**Comment by skovaka** [comment]
Sorry about that, this should now be fixed in the most recent commit (tagged v1.1). The previous version was written for the previous version of the read_until_api which required a developer licence, but I've now updated everything to work with the newly public version. Let me know if you have any further issues.

**Comment by vincentmanz** [comment]
Brilliant, working now.
